WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -500,6 +504,34 @@ extension ShippingLabelFormViewModel {
         case validationError(ShippingLabelAddressValidationError)
         case genericError(Error)
     }
+
+    func purchaseLabel(onCompletion: @escaping ((Bool) -> Void)) {
```

Comment:
What do you think about returning `Result<Void, Error>` that way we can switch over a semantic meaning(success, error) rather than a boolean that can mean anything! 

Also since this method is public it would be nice to add some docs to it!